pyanaconda/modules/storage/storage_interface.py

```diff
@@ -40,7 +40,7 @@ def connect_signals(self):
             "AppliedPartitioning", self.implementation.applied_partitioning_changed
         )
 
-    def ScanDevicesWithTask(self) -> ObjPath:
+    def ScanDevicesWithTask(self, deep_scan: bool = True) -> ObjPath:
```

Comment:
I would prefer a new method for this that can be marked as a temporary workaround and eventually dropped.